.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```

Comment:
Good change.  Was going to suggest.

Reply:
Yeah I completely forgot I had used CI here instead of install before